spharpy/classes/coordinates.py

```diff
@@ -511,28 +594,20 @@ def weights(self):
     @weights.setter
     def weights(self, weights):
         r"""The area/quadrature weights of the sampling.
-        Their sum must equal to :math:`4\pi`.
+        Their sum must equal to :math:`4\pi`. If the provided
+        weights do not fulfill the quadrature condition,
+        the quadrature flag will be updated accordingly.
```

Comment:
Should be identical for getter and setter. I would omitt the information about the quadrature flag.

```suggestion
        r"""Get or set the area/quadrature weights of the sampling.
        Their sum must equal to :math:`4\pi`.
```